### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by marcoow**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,47 +21,44 @@
   },
   "dependencies": {
     "ember-cli-babel": "^5.1.7",
-    "ember-getowner-polyfill": "^1.2.2"
+    "ember-getowner-polyfill": "^1.1.1"
```
Comment (Turbo87) [inline package.json]: why are we downgrading?

File: package.json [hunk 2]
```diff
@@ -21,47 +21,44 @@
...
   },
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.5",
     "chai": "^3.5.0",
-    "chalk": "^1.1.1",
+    "chalk": "^1.1.3",
+    "broccoli-asset-rev": "^2.4.5",
```
Comment (Turbo87) [inline package.json]: wrong sort order

File: package.json [hunk 3]
```diff
@@ -21,47 +21,44 @@
...
     "ember-ajax": "^2.4.1",
-    "ember-cli": "2.11.0",
-    "ember-cli-addon-tests": "^0.6.0",
-    "ember-cli-app-version": "^2.0.0",
+    "ember-cli": "2.12.0",
     "ember-cli-chai": "^0.3.2",
+    "ember-cli-addon-tests": "^0.6.3",
```
Comment (Turbo87) [inline package.json]: wrong sort order

File: package.json [hunk 4]
```diff
@@ -21,47 +21,44 @@
...
     "ember-resolver": "^2.0.3",
-    "ember-sinon": "0.6.0",
-    "ember-source": "^2.11.0",
-    "eslint-config-simplabs": "0.3.1",
-    "eslint-plugin-ember": "^2.0.1",
-    "eslint-plugin-mocha": "^4.8.0",
-    "glob": "^7.0.3",
-    "loader.js": "^4.0.10",
+    "ember-source": "~2.12.0",
+    "ember-sinon": "^0.6.0",
```
Comment (Turbo87) [inline package.json]: wrong sort order

btw https://github.com/keithamus/sort-package-json 😉

File: package.json (hunk 1 above)
Comment (marcoow) [inline package.json]: We can actually work with 1.1.1 as well and to allow for better deduplication with yarn I think we should specify the dependency version as relaxed as possible.

**Comment by marcoow** [comment]
@Turbo87: sorted the `package.json`

**Comment by marcoow** [comment]
@Turbo87: should be fine now

**Pull request merged**